examples/with_winit/src/lib.rs

```diff
@@ -655,16 +663,36 @@ use winit::platform::android::activity::AndroidApp;
 #[no_mangle]
 fn android_main(app: AndroidApp) {
     use winit::platform::android::EventLoopBuilderExtAndroid;
-
-    android_logger::init_once(
-        android_logger::Config::default().with_max_level(log::LevelFilter::Warn),
-    );
-
+    let config = android_logger::Config::default();
+    // We allow configuring the Android logging with an environment variable at build time
+    let config = if let Some(logging_config) = option_env!("VELLO_STATIC_LOG") {
+        let mut filter = env_logger::filter::Builder::new();
+        filter.filter_level(log::LevelFilter::Warn);
+        filter.parse(logging_config);
+        let filter = filter.build();
+        // This shouldn't be needed in theory, but without this the max
+        // level is set to 0 (i.e. Off)
+        let config = config.with_max_level(filter.filter());
+        config.with_filter(filter)
+    } else {
+        config.with_max_level(log::LevelFilter::Warn)
+    };
+    android_logger::init_once(config);
+    let cache_dir = get_cache_directory(&app)
```

Comment:
This will be used for pipeline caching, but that isn't yet implemented. I could move this to a different PR, but it would be useful for me to have it merged, and I don't think there's any real cost to keeping it here (especially with #433, if with_winit is to be more of a testbed)